exercises/sum-of-multiples/sum-of-multiples.spec.js

```diff
@@ -2,23 +2,23 @@ var SumOfMultiples = require('./sum-of-multiples');
 
 describe('SumOfMultiples', function () {
   it('to 1', function () {
-    expect(SumOfMultiples().to(1)).toBe(0);
+    expect(SumOfMultiples([3,5]).to(1)).toBe(0);
```

Comment:
I know it's super minor, but it's possible this could be `[3, 5]` with an extra space if want to be consistent with the styles of all below lists, which do have spaces after their commas.